.okta/sample-config.yaml

```
@@ -6,7 +6,7 @@ oauthClient:
   applicationType: native
 
 directions: |+2
-  Okta configuration written to: app/src/main/res/raw/okta_oidc_config.json
+  Okta configuration written to: okta.properties
   Run instructions for Android studio:
     • configure an Android virtual device (emulator) or connect a real device to Android Studio
-    • run the application using 'Shift+F10' on Windows/Linux, 'Control+R' on macOS, or click 'Run' button in the toolbar.
\ No newline at end of file
+    • run the application using 'Shift+F10' on Windows/Linux, 'Control+R' on macOS, or click 'Run' button in the toolbar.
```

I'd suggest a slight update to follow Okta style (I capitalized Android Studio as it's a copyrighted term, removed Android Studio from the bullet points as it's clear from context, removed the command-key equivalents as those are easily discoverable... besides Studio should be teaching those shortcuts.)

```suggestion
Run the app in Android Studio:
- Create a target for the app by either configuring a Android virtual device (emulator) or connecting a physical device.
- Run the app by clicking the Run button in the toolbar.
```